project 2/styles.css

```
@@ -0,0 +1,154 @@
+* {
+    box-sizing: border-box;
+}
+
+h1 {
+    font-family: 'Big Shoulders Display', cursive;
+    font-weight: 900;
+    text-align: center;
+    font-size: 24.87pt;
+    letter-spacing: 1px;
+    margin-top: 40px;
+    color: #32302A;
+}
+
+
+ 
+ ul.info {
+    font-family: 'Big Shoulders Display', cursive;
+    font-weight: 800;
+    font-size: 12pt;
+    border-top: 4px solid #CCC1A9;
+    border-bottom: 4px solid #CCC1A9;
+    border-left: 4px solid #CCC1A9;
+    display:inline-block;
+    list-style: none;
+    padding-left: 0;
+    margin-top: 1px;
+    margin-right: 15px;
+}
+li.info   {
+    border-right: 4px solid #CCC1A9;
+    display: inline-block;
+    padding: 0px 23px;
+  }
+li.special  {
+    
+    display: inline-block;
+    padding: 0px 15px;
+    
+  }
+  
+li.special + li.info {
+    border-left: 4px solid #CCC1A9;
+    padding: 0.1px 15px;
+    
+  }
+
+.special {
+    font-weight: 500;
+    
+}
+img {
+    width: 124px;
+    height: 179px;
+    border-radius: 0.25in;
+    display: inline-block; 
+    position: relative;
+    left: -37%; 
+    bottom: -23px;
+    
+}
+/* image source: ­­­­https://www.xn--sommerkinorntgenplatz-rec.ch/index.php/events */
+.text{
+    display: inline-block;
+    position: relative;
+    right: -87px;
+    top: -215px; 
+}
+.content {
+        background-color: #FDF1DB;
+        border-radius: 25px;
+        height: 225px;
+        width: 600px;
+        margin-right: auto;
+        margin-left: auto;
+        line-height: 2;
+        text-align: center;
+        position: relative;
+        left:auto;
+        top: 15px; 
+        color: #32302A;
+      
+   
+        
+}
+
+section {
+    width: 640px; 
+    background-color: #CCC1A9;
+    height: 256px; 
+    margin-right: auto;
+        margin-left: auto;
+        box-shadow: 0px 9.6px 9.6px grey;
+}
+
+#line {
+    height: 163.2px;
+    position: absolute;
+    top: 30px;
+    left: 155px;
+}
+
+
+span {
+    font-weight: 600; 
+    font-size: 10pt;
+}
+
+.watch {
+    font-family: 'Big Shoulders Display', cursive;
+    font-weight: 500;
+    font-size: 12pt;
+    letter-spacing: 1px;
+    margin-top: 0px;
```

Same note about not using units on zero  measurements.
```suggestion
    margin-top: 0;
```